docs/requirements.txt

```diff
@@ -1,3 +1,6 @@
 sphinx
 sphinx_rtd_theme
 sphinx-autoapi
+
+kivy
+Pillow
```

Comment:
These are actually requirements of the library, not the docs. These should be moved to `PluginPlayer/requirements.txt`.